MapleServer2/Types/FieldState.cs

```diff
@@ -233,4 +233,29 @@ public void AddVibrateObject(MapVibrateObject vibrateObject)
     {
         VibrateObjects[vibrateObject.EntityId] = vibrateObject;
     }
+
+    public IFieldActor GetActor(int actorId)
+    {
+        if (Players.TryGetValue(actorId, out Character player))
+        {
+            return player;
+        }
+
+        if (Mobs.TryGetValue(actorId, out Npc mob))
```

Comment:
perhaps using lambdas is cleaner here, =>

Reply:
What did you have in mind for a change here? I'm still getting used to C# so I'm not sure what the proposed change here is

Reply:
oh god, nono ignore it, I thought it was a function, 2 hours of sleep checks out.
Sorry.